src/main/resources/site/parts/market-latest/market-latest.html

```diff
@@ -0,0 +1,17 @@
+<div th:if="${hasCachedItems}" class="market_latest">
```

Comment:
Do a conditional remove tag if we don't want to render the topLevel div. 
Just add it to the modal when rendering the output.
```suggestion
<div th:if="${hasCachedItems}" th:remove="${removeTop ? 'tag' : ''}" class="market_latest">
```

Reply:
Such a nice approach! Thank you Persijn, I'll do that :)